main.py

```diff
@@ -0,0 +1,41 @@
+import random
+from hangman_words import word_list
+from hangman_art import logo, stages
+from replit import clear
+
+def play_hangman():
```

Comment:
We have encapsulated the main game logic in a function called `play_hangman()`, which makes the code more organized and easier to read.

We use a list called `guessed_letters` to keep track of letters that have already been guessed, so that we can give a prompt if the player repeats a guess.

Furthermore, we simplified some lines of code, combining the initial value assignment of `display` and the `for` loop into a list comprehension. We changed `while not end_of_game` to `while True` loop and used `break` to exit the loop.